pages/linux/conky.md

```diff
@@ -0,0 +1,27 @@
+# conky
+
+> Light-weight system monitor for X.
+
+- Launch with default, built-in config:
+
+`conky`
+
+- Create a new, default config:
+
+`conky -C > ~/.conkyrc`
+
+- Load a config file:
+
+`conky -c {{/path/to/config}}`
+
+- Start in the background (daemonize):
+
+`conky -d`
+
+- Align specified config file on screen:
```

Comment:
This is a bit unclear to me. What does it mean to align a config file on screen ?

Reply:
It's to align the display on the desktop. So for example, `conky -a top_right` will place the system monitor on the top-right corner of the desktop.